server/controllers/comment.controller.js

```diff
@@ -22,22 +51,41 @@ function create(req, res, next) {
   comment.author = user._id
   comment.save()
   .then((commentSaved)  => {
+    // TODO: result key is not consistent with other responses, consider changing this
```

Comment:
Ah yeah.  My thinking was doing it this way could allow us to add meta data later on.  For example 'cursors' when doing list type requests.  Sadly it's inconsistent and provides no benefit right now.

Reply:
It seemed like a good idea but I was wondering about what the best practice was. Digital Ocean is often lauded for the amazing API and they seem to do [something similar](https://developers.digitalocean.com/documentation/v2/#list-all-droplets). If you think it's a good idea, we should keep it and a long-term todo refactor could be migrating everything else to be consistent with this idea.

Reply:
Why don't we just use the standard data key return? We can always add other meta data as well.

Reply:
I am not sure what you mean?  Either way, it's probably not a big deal to have the comments endpoint  match the other ones.